seacrowd/sea_datasets/phoatis/phoatis.py

```diff
@@ -0,0 +1,236 @@
+                "https://raw.githubusercontent.com/VinAIResearch/JointIDSF/main/PhoATIS/word-level/test/seq.in",
+                "https://raw.githubusercontent.com/VinAIResearch/JointIDSF/main/PhoATIS/word-level/test/seq.out",
+                "https://raw.githubusercontent.com/VinAIResearch/JointIDSF/main/PhoATIS/word-level/test/label",
+            ],
+        },
+    }
+}
+
+_SUPPORTED_TASKS = [Tasks.INTENT_CLASSIFICATION, Tasks.SLOT_FILLING]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+def config_constructor_intent_cls(schema: str, version: str, phoatis_subset: str = "syllable") -> SEACrowdConfig:
+    assert phoatis_subset == "syllable" or phoatis_subset == "word"
+
+    return SEACrowdConfig(
+        name="phoatis_intent_cls_{phoatis_subset}_{schema}".format(phoatis_subset=phoatis_subset.lower(), schema=schema),
+        version=version,
+        description="PhoATIS Intent Classification: {subset} {schema} schema".format(subset=phoatis_subset, schema=schema),
+        schema=schema,
+        subset_id=phoatis_subset,
+    )
+
+
+def config_constructor_slot_filling(schema: str, version: str, phoatis_subset: str = "syllable") -> SEACrowdConfig:
+    assert phoatis_subset == "syllable" or phoatis_subset == "word"
+
+    return SEACrowdConfig(
+        name="phoatis_slot_filling_{phoatis_subset}_{schema}".format(phoatis_subset=phoatis_subset.lower(), schema=schema),
+        version=version,
+        description="PhoATIS Slot Filling: {subset} {schema} schema".format(subset=phoatis_subset, schema=schema),
+        schema=schema,
+        subset_id=phoatis_subset,
+    )
+
+
+class PhoATIS(datasets.GeneratorBasedBuilder):
+    """This is first public intent detection and slot filling dataset for Vietnamese. The data contains 5871 English utterances from ATIS that are manually translated by professional translators into Vietnamese."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [config_constructor_intent_cls("source", _SOURCE_VERSION, subset) for subset in ["syllable", "word"]]
+    BUILDER_CONFIGS.extend([config_constructor_intent_cls("seacrowd_text", _SEACROWD_VERSION, subset) for subset in ["syllable", "word"]])
+    BUILDER_CONFIGS.extend([config_constructor_slot_filling("source", _SOURCE_VERSION, subset) for subset in ["syllable", "word"]])
+    BUILDER_CONFIGS.extend([config_constructor_slot_filling("seacrowd_seq_label", _SEACROWD_VERSION, subset) for subset in ["syllable", "word"]])
+
+    BUILDER_CONFIGS.extend(
+        [  # Default config
+            SEACrowdConfig(
+                name="phoatis_source",
+                version=SOURCE_VERSION,
+                description="PhoATIS Intent Classification source schema (Syllable version)",
+                schema="source",
+                subset_id="syllable",
+            ),
+            SEACrowdConfig(
+                name="phoatis_seacrowd_text",
```

Comment:
Can we use `"phoatis_intent_cls_seacrowd_text"` instead and adjust the codebase accordingly?

Reply:
same comment as the reply below.